lib/brain.js

```diff
@@ -2043,8 +2042,8 @@ Brain.prototype.smsCompliance = function smsCompliance (opts = {}) {
 }
 
 Brain.prototype.validateTier = function validateTier (tier, triggerTx, customerTier, requiredTiers) {
-  const requiredManualTiers = _.filter(tier => _.get(tier)(this.trader.triggersAutomation) === MANUAL)(requiredTiers)
-  const scannedManualTiers = _.filter(tier => _.get(tier)(this.manualTriggersDataProvided))(requiredManualTiers)
+  const requiredManualTiers = _.filter(tier => !_.get(tier, this.trader.triggersAutomation), requiredTiers)
```

Comment:
Were changes regarding this `MANUAL` variable done on the server side? Upon a quick look, didn't find any changes regarding "compensation" to this one, so just asking in case I missed it

Reply:
The old poll handler on the server still replies with `Manual` or `Automatic`, for backwards compatibility. But the old reply handler on the machine makes that a boolean, as you saw.

Reply:
I'm not sure I prefer to have it this way, any reason for the swap from AUTOMATIC|MANUAL to a boolean?